.github/workflows/scripts/linkerd_deploy.sh

```diff
@@ -8,16 +8,18 @@
 export MESH_NAME='Linkerd'
 export SERVICE_MESH='LINKERD'
 
-curl -fsL https://run.linkerd.io/install | sh
-export PATH=$PATH:/home/runner/.linkerd2/bin
-linkerd version
-linkerd check --pre
-linkerd install | kubectl apply -f -
-linkerd check
+# Check if mesheryctl is present, else install it
+if ! [ -x "$(command -v mesheryctl)" ]; then
+    echo 'mesheryctl is not installed. Installing mesheryctl client... Standby... (Starting Meshery as well...)' >&2
+    curl -L https://meshery.io/install | ADAPTERS=linkerd PLATFORM=kubernetes bash -
+fi
 
-curl -fsL https://run.linkerd.io/emojivoto.yml | kubectl apply -f -
-kubectl -n emojivoto port-forward svc/web-svc 8080:80 &> /dev/null &
-kubectl get -n emojivoto deploy -o yaml | linkerd inject - | kubectl apply -f -
+curl -fsL https://run.linkerd.io/emojivoto.yml 
```

Comment:
The curl command isn't putting the output in a file. For reference :- https://github.com/hershd23/meshery-smp-action/runs/6868074975?check_suite_focus=true#step:4:285

We can add it explicitly like shown

```suggestion
curl -fsL https://run.linkerd.io/emojivoto.yml --output emojivoto.yml
```